Created form redirect guide (#945)
blutui/dev.blutui.com: content/guides/add-redirect-to-form.mdx

---
title: How do I redirect after submitting a form?
description: Learn how to automatically redirect users to a thank-you or confirmation page after submitting a form in Blutui.
---

## Introduction

When a user submits a form, you may want to redirect them to a **thank-you page**, **confirmation screen**, or any other route in your project.  
Blutui makes this easy — simply add a hidden `redirect` field inside your form.

This guide will show you how to set up a form redirect in just a few lines.

---

### 1. Wrap your form with the form tags

First, ensure your form is created using the Canvas form tag:

```canvas
{% form 'contact' %}
  
{% endform %}
````

> Replace `'contact'` with the handle of the form you created in the dashboard.

Reviewer: ```suggestion
Replace `contact` with the handle of the form you created in the dashboard.
```

> If you need help creating a form, see:

Reviewer: ```suggestion
If you need help creating a form, view the [How do I make a form in a Blutui project?](/guides/create-form) guide.
```

[How do I make a form in a Blutui project?](https://dev.blutui.com/guides/create-form)

---

### 2. Add the redirect field

Inside your `{% form %}` block, add a hidden input named `redirect`:

```canvas
{% form 'contact' %}

  <input type="hidden" name="redirect" value="/contact/thank-you">

{% endform %}
```

The `value` is the page you want the user to be taken to after the form is submitted.

In this example:

```
/contact/thank-you
```

This means the user will be redirected to the **thank-you page** once the form is processed.

---

### 3. Make sure the redirect page exists

The redirect will only work if the target page exists.

If you need help creating a page, follow this guide:

> [How do I create a page?](https://dev.blutui.com/guides/create-page)

Reviewer: ```suggestion
```
